boost-configure

```diff
@@ -0,0 +1,11 @@
+
+./bootstrap.sh "$@" --without-libraries=atomic,chrono,container,context,coroutine,coroutine2,date_time,exception,filesystem,graph,graph_parallel,iostreams,locale,log,math,mpi,program_options,python,random,regex,serialization,signals,system,test,thread,timer,type_erasure,wave
```

Comment:
Personally I prefer listing libraries to build (`--with-libraries`) instead of excluding not wanted ones.